src/test/java/pages/ProfilePersonalInfoPage.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ProfilePersonalInfoPage {
+    private final WebDriver driver;
+
+    public ProfilePersonalInfoPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    @FindBy(xpath = "//*[text()='My Profile']")
+    private WebElement myProfileBtn;
+
+    @FindBy(xpath = "//*[text()='Current Orders']")
+    private WebElement currentOrdersBtn;
+
+    @FindBy(xpath = "//*[text()='Order History']")
+    private WebElement orderHistoryBtn;
+
+    public void myProfileAccess() {
+        myProfileBtn.click();
+    }
+    public void currentOrdersAccess() {
+        currentOrdersBtn.click();
+    }
+    public void orderHistoryAccess() {
+        orderHistoryBtn.click();
+    }
+}
```

Comment:
Maybe we should return instance of page in this methods, since we have Page Objects for them?

Reply:
If we return instance of page ProfileOrderHistoryPage in current page, we will violate the SOLID principle.